### ISSUE TITLE
Understanding document for “Dragging”, a non-existent WCAG 2.2 SC, is still available

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#1835]
The [“Dragging” SC](https://www.w3.org/WAI/WCAG22/Understanding/dragging.html) has been replaced by the [“Dragging Movement” SC](https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements) at some point.

The understanding page for Dragging is still available, despite the page having been updated significantly. People find the Dragging page and get confused (this just happened in the a11y slack). Please delete pages that are not relevant anymore.

Dragging Understanding: https://www.w3.org/WAI/WCAG22/Understanding/dragging.html
Dragging Movements Understanding: https://www.w3.org/WAI/WCAG22/Understanding/dragging-movements

Related to #1833

**Comment by alastc** [comment]
The old version isn't in the source code, so it must be a publishing issue, so assigning to @michael-n-cooper

**Comment by yatil** [comment]
I assume that the files are uploaded using CVS. And CVS does not delete things on the server unless explicitly instructed to do so.

**Comment by shawna-slh** [comment]
I set a redirect for now. (since Michael is not available for a while, afaik)

Please leave this issue open for Michael to address as he wishes when he gets to it.

@alastc  Please let me know if there are others to redirect for now.

**Comment by alastc** [comment]
Closing as you can't access the old version anymore. @michael-n-cooper I'm not sure if there's anything you need to do?
